Signum.Entities.Extensions/Dashboard/DashboardEntity.cs

```diff
@@ -264,4 +266,18 @@ public enum DashboardEmbedededInEntity
     Tab
 }
 
+[InTypeScript(true)]
+public enum PanelStyle

```

Comment:
Remove PanelStyle and add only string? CustomColor, keep BootstrapStyle for Workflow